### DISCUSSION TITLE
can I access older images?

### DISCUSSION THREAD

**Discussion by Solarbytes** [johang/sd-card-images#86] [General]
Hi there,

First of all, thank you very much for making those pcduino images available, it saved my board from the scrap pile!

Back in July, I downloaded an image and was able to make teslausb work with the pcduino 3 nano. Something failed in my DD of the working image and I did not check after making it - my bad...

I tried the latest image but I keep running into system errors "BUG: scheduling while atomic" or something similar.

Is there an archive of the older images so I can try with the older one to confirm whether it's the OS or the installation script that has changed/is causing the issue? It was the bookworm image from July (late july, downloaded around 26th July apparently)

**Comment by johang** [comment]
I don't keep an archive of old images.

"BUG: scheduling while atomic" sounds like a driver issue somewhere. Maybe you can try the other images (Bullseye, Trixie, Sid), use a different kernel.
